tools/test-ecosystem.mjs

```diff
@@ -0,0 +1,154 @@
+/**
+ * @fileoverview A utility to test an ecosystem plugin against the built ESLint.
+ * @author Josh Goldberg
+ */
+
+"use strict";
+
+import chalk from "chalk";
+import { $ } from "execa";
+import fs from "node:fs/promises";
+import path from "node:path";
+import util from "node:util";
+
+const plugins = new Map([
+	[
+		"eslint-plugin-unicorn",
+		{ repository: "https://github.com/sindresorhus/eslint-plugin-unicorn" },
+	],
+	["@eslint/css", { repository: "https://github.com/eslint/css" }],
+	["@eslint/json", { repository: "https://github.com/eslint/json" }],
+	["@eslint/markdown", { repository: "https://github.com/eslint/markdown" }],
+	[
+		"@eslint-community/eslint-plugin-eslint-comments",
+		{
+			repository:
+				"https://github.com/eslint-community/eslint-plugin-eslint-comments",
+		},
+	],
+	[
+		"eslint-plugin-vue",
+		{ repository: "https://github.com/vuejs/eslint-plugin-vue" },
+	],
+	[
+		"typescript-eslint",
+		{
+			linker: `na link ${process.cwd()}`,
```

Comment:
[Question] I don't love having to manually specify this. Since `ni` doesn't have one itself (https://github.com/eslint/rfcs/pull/127/files#diff-7c99be846e07d8842a67527bcd47eb3175446759ac12beb6db87ac712a00a1f9R41) I'm tempted to publish a standalone script too, named `ni-link` or something like that. Does anybody know of an existing equivalent though?

Reply:
Moot point now that we no longer have typescript-eslint in the initial list.